### ISSUE TITLE
As a team leader I want to see a leaderboard of who has the most holiday left so I can ensure that all holiday is used

### ISSUE THREAD

**Issue by stevenhaynes** [innovationprogrammeuki/Absences#10]
AC01: Does the dashbard show a leaderboard capped at 15 ranking who has the most holiday at the top to who has the least holiday at the bottom?

**Comment by danieljtate** [comment]
Working on cleaning up code for code review

**Comment by JohnThirlaway1** [comment]
I am currently integrating daniels code into my story as the two stories will be displayed on the same webpage

**Comment by JohnThirlaway1** [comment]
Code has been cleaned up by myself and Daniel, and peer reviewed by Ryan. Currently waiting on Michael story to be approved as the base code in-order to push up to gerit  for code review.

**Comment by danieljtate** [comment]
Code has been pushed for review

**Comment by danieljtate** [comment]
Working on making the changes highlighted within code review

**Comment by JohnThirlaway1** [comment]
story has been code reviewed as part of story point 9 and has passed, will be merged into master once a few small changes have been made after the UI review.
